code/game/machinery/shieldgen.dm

```diff
@@ -393,9 +393,9 @@
 	return ..()
 
 
-/obj/machinery/power/shieldwallgen/wrench_act(mob/living/user, obj/item/I)
+/obj/machinery/power/shieldwallgen/wrench_act(mob/living/user, obj/item/tool)
 	. = ..()
-	. |= default_unfasten_wrench(user, I, 0)
+	. |= default_unfasten_wrench(user, tool, time = 0)
```

Comment:
```suggestion
	. |= default_unfasten_wrench(user, tool)
```

Reply:
This would make them have a 2 second timer, which I don't want.

Reply:
0 SECONDS?

Reply:
> 0 SECONDS?

yes some items are meant to unfasten instantly, lockers for example wrench and unwrench instantly, a couple machines have also been instant for quite awhile, that said maybe none of these SHOULD be instant despite that being how they worked forever